File: app/src/main/java/model/WriteDataResponse.java
package model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WriteDataResponse {
    @SerializedName("gravados")
    @Expose
    int savedGames;

    public int getSavedGames() {
        return savedGames;
    }

    public int getTotal() {
        return total;
    }

    @SerializedName("total")
    @Expose
    int total;

    public WriteDataResponse(int savedGames, int total) {
        this.savedGames = savedGames;
        this.total = total;
    }
}

File: app/src/main/kotlin/model/WriteDataResponse.kt
package model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class WriteDataResponse(
        @SerializedName("gravados")
        @Expose
        val savedGames: Int,

        @SerializedName("total")
        @Expose
        val total: Int
)
